.browserslistrc

```diff
@@ -1,3 +1,3 @@
 > 1%
 last 2 versions
-not ie <= 10
+not dead
```

Comment:
I changed this otherwise, Vue will compile a legacy package version and a modern package version. Let me know if this is ok.

Reply:
Yeah, we don't support IE11 or earlier.